Notifications/notifications_actions_customization.md

```diff
@@ -11,100 +11,118 @@ This document is a starting point for engaging the community and standards bodie
 ## Introduction
```

Comment:
For the images above, would it look better if we used white icons with the colored buttons to match the white text?

Reply:
Actually the icon image I used is white and the Windows Notification API is making it black when used in light theme. When I switch to the dark theme, the icons become white. The [official docs](https://learn.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/adaptive-interactive-toasts?tabs=xml#buttons-with-colors) page does not mention anything about this. I will keep the images like that.